.github/workflows/ghcr.yml

```diff
@@ -1,8 +1,8 @@
-name: Build and publish multi-arch container images
+name: Publish Docker Image
 
 on:
   push:
-    branches: [ main ]
+    branches: [ '**' ]
```

Comment:
At 15+ mins per build, I'm not a fan of "build every commit"
It doesn't block PR merges right?

Reply:
Yeah agree. I'm hopeful some kind of caching will speed this up, but that's a future issue.

It won't block PRs for now.